.azure/azure-pipelines.ci.yml

```diff
@@ -198,6 +198,28 @@ stages:
 # Performance Tests
 #
 
+- stage: remoteperformance
+  displayName: Remote Performance Testing
+  dependsOn:
+  - build_windows
+  jobs:
+  - template: ./templates/run-remote-performance-int.yml
```

Comment:
Let's get rid of the `-int`. I just refactored most of them out with my last PR.